cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -221,6 +226,12 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
         'https://{{.Name}}.${containerAppsEnvironment.outputs.defaultDomain}'
         {{- end}}
       ]
+      allowedMethods: [
+        'GET'
+        'PUT'
+        'POST'
+        'DELETE'
+      ]
```

Comment:
Without this, the frontend app can not do write / delete operation.

Reply:
I see; it's likely that the allowedMethods is just `GET` `PUT` `POST` as the default. 

Does `*` work? I think based on these [docs](https://learn.microsoft.com/en-us/azure/container-apps/cors?tabs=arm&pivots=azure-portal#enable-and-configure-cors) it should.

Otherwise, we may have to worry about `HEAD` requests that seem to be common these days with browser pre-flight checks.

Reply:
Updated.